.github/workflows/github.yml

```diff
@@ -145,6 +145,13 @@ jobs:
       with:
         submodules: recursive
 
+    - name: Validate JSON
+      # the Python yaml module doesn't seem to work on mac-arm
+      if: ${{ startsWith(matrix.preset, 'linux') }}
```

Comment:
Running it on multiple presets is probably redundant and slightly increases already long CI built times.
Perhaps run only on `[build (linux-qt6, ubuntu-22.04, 0, linux-clang-test)]`?

This is our fastest Linux-based CI preset